src/DataContainer/DcaFilterInterface.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/*
+ * This file is part of Contao.
+ *
+ * Copyright (c) 2005-2017 Leo Feyer
+ *
+ * @license LGPL-3.0+
+ */
+
+namespace Contao\CoreBundle\DataContainer;
+
+/**
+ * DCA filter interface.
```

Comment:
Yeah that would be fine. It's just that if you see this interface and you never came across it you won't have no idea about why it's there and what it does.